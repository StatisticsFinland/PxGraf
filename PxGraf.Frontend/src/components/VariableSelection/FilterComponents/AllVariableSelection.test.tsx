import { render } from "@testing-library/react";
import React from "react";
import AllVariableSelection from "./AllVariableSelection";

jest.mock('react-i18next', () => ({
    ...jest.requireActual('react-i18next'),
    useTranslation: () => {
        return {
            t: (str: string) => str,
            i18n: {
                changeLanguage: () => new Promise(() => { }),
            },
        };
    },
}));

describe('Rendering test', () => {
    it('renders correctly', () => {
        const { asFragment } = render(<AllVariableSelection
        ></AllVariableSelection>);
        expect(asFragment()).toMatchSnapshot();
    });
});