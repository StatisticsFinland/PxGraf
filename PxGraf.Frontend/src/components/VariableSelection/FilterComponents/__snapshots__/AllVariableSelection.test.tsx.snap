// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering test renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-aXZVg hOKDeU MuiBox-root css-0"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      variableSelect.allFilter
    </p>
  </div>
</DocumentFragment>
`;
